grafana-plugin/src/plugin/GrafanaPluginRootPage.tsx

```diff
@@ -105,22 +118,60 @@ export const Root = observer((props: AppRootProps) => {
         })}
       >
         {userHasAccess ? (
-          <Page {...props} query={...getQueryParams()} path={pathWithoutLeadingSlash} store={store} />
+          <Switch>
+            <Route path={getRoutesForPage('incidents')} exact>
+              <Incidents query={query} />
+            </Route>
+            <Route path={getRoutesForPage('incident')} exact>
+              <Incident query={query} />
+            </Route>
+            <Route path={getRoutesForPage('users')} exact>
+              <Users query={query} />
+            </Route>
+            <Route path={getRoutesForPage('integrations')} exact>
+              <Integrations query={query} />
+            </Route>
+            <Route path={getRoutesForPage('escalations')} exact>
+              <EscalationChains />
+            </Route>
+            <Route path={getRoutesForPage('schedules')} exact>
+              <Schedules />
+            </Route>
+            <Route path={getRoutesForPage('schedule')} exact>
+              <Schedule />
+            </Route>
+            <Route path={getRoutesForPage('outgoing_webhooks')} exact>
+              <OutgoingWebhooks />
+            </Route>
+            <Route path={getRoutesForPage('maintenance')} exact>
+              <Maintenance query={query} />
+            </Route>
+            <Route path={getRoutesForPage('settings')} exact>
+              <SettingsPage />
+            </Route>
+            <Route path={getRoutesForPage('organization-logs')} exact>
+              <OrganizationLogPage />
+            </Route>
+            <Route path={getRoutesForPage('chat-ops')} exact>
+              <ChatOps />
+            </Route>
+            <Route path={getRoutesForPage('live-settings')} exact>
+              <LiveSettings />
+            </Route>
+            <Route path={getRoutesForPage('cloud')} exact>
+              <CloudPage />
+            </Route>
+            <Route path={getRoutesForPage('test')} exact>
+              <Test />
+            </Route>
+            <Route path="*">
+              <NoMatch />
+            </Route>
```

Comment:
The route for `?page=incident` was changed to `/incidents` and older links no longer let you view an incident. You can try by this link to see if it works for you `http://localhost:3000/a/grafana-oncall-app/?id=[ID_HERE]&page=incident&perpage=25&start=1`

It redirects to `/incidents` - it would be nice if we could redirect all older links to newer links